outlines/models/llamacpp.py

```diff
@@ -288,6 +289,16 @@ def llamacpp(
     if "verbose" not in llamacpp_model_params:
         llamacpp_model_params["verbose"] = False
 
+    # TODO: Remove when https://github.com/ggerganov/llama.cpp/pull/5613 is resolved
+    if "tokenizer" not in llamacpp_model_params:
+        warnings.warn(
+            "llama.cpp pre-tokenizer is broken. "
```

Comment:
```suggestion
            "The pre-tokenizer in `llama.cpp` is broken. "
```

+ link to corresponding issue in `llama.cpp` for context.